### PR TITLE
Replace button with input + change color

### PR DESCRIPTION
**Pull request by freewill777**
This Pull Request brings the changes necessary to:
- replace the button with an input in the Banner section of the app.
- change banner color to green

New Note

### CODE REVIEW

File: src/Banner.jsx [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import React from "react"
-import Button from '@mui/material/Button';
 import Box from '@mui/material/Box';
+import InputBase from '@mui/material/InputBase';
 
 function Banner() {
     const [color, setColor] = React.useState('#457b9d')
```
Comment (andradaVoinea) [inline src/Banner.jsx]: Can we change default color?

File: src/Banner.jsx [hunk 2]
```diff
@@ -20,16 +20,17 @@ function Banner() {
...
             }}
         >
-            <Button
-                onClick={handleClick}
-                variant="outlined"
-                style={{
+            <InputBase
+                sx={{
+                    ml: 1,
+                    p: 1,
```
Comment (freewill777) [inline src/Banner.jsx:26]: ??

**Review by andradaVoinea — APPROVED** [review]
OK

File: src/Banner.jsx (hunk 1 above)
Comment (freewill777) [reply src/Banner.jsx]: done

**Review by Cladiu — APPROVED** [review]

**Review by Cladiu — APPROVED** [review]

**Pull request merged**